packages/ui/src/components/ContentReference/ChatReference.tsx

```diff
@@ -44,6 +44,7 @@ export default function ChatReference({
           <ContactName
             color="$tertiaryText"
             size="$s"
+            maxWidth={230}
```

Comment:
Would probably be good to leave a comment explaining why this specific number is used.